solr/CHANGES.txt

```diff
@@ -31,6 +31,9 @@ Jetty 9.4.19.v20190610
 Upgrade Notes
 ----------------------
 
+* SOLR-13207: Queries that use the < operator not followed by a number will now return an HTTP 400 error rather than 500,
```

Comment:
This should be in the 8.4/"Bug Fixes" section (I plan to backport). I would also mention that this fix is specific to minimum should match in dismax/edismax query parsers

Reply:
Done (21b448e, just pushed).